app/src/main/java/org/mozilla/rocket/widget/NewsSourcePreference.java

```diff
@@ -9,16 +9,18 @@
 import android.support.annotation.StringDef;
 import android.util.AttributeSet;
 import android.util.Log;
+
 import org.mozilla.focus.utils.AppConfigWrapper;
 import org.mozilla.focus.utils.Settings;
 import org.mozilla.rocket.content.NewsSourceManager;
 
 public class NewsSourcePreference extends ListPreference {
 
+    // "DainikBhaskar.com" doesn't provide their feed anymore.
     public final static String NEWS_DB = "DainikBhaskar.com";
     public final static String NEWS_NP = "Newspoint";
 
-    @StringDef({NEWS_DB, NEWS_NP})
+    @StringDef({NEWS_NP})
```

Comment:
This annotation is not used at all, consider using it or removing it. Future addition of this annotation may lead to unwanted crash if user is migrating from before and after the addition of this annotation.